dummies/protoss/proxy_zealot_rush.py

```diff
@@ -32,6 +55,9 @@ async def start(self, knowledge: "Knowledge"):
         self.gather_point = self.pather.find_path(self.proxy_location, self.knowledge.enemy_start_location, 8)
 
     async def build_order(self):
+        my_mgr: MyManager = self.knowledge._managers_dict.get(MyManager)
```

Comment:
I would like to provide a method for getting a manager, instead of accessing the dict directly.